src/Classes/DiskSpace.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Classes;
+
+/**
+ * Container for PHP Builtin disk space functions that is easy to mock in tests
```

Comment:
_superlike!_

Reply:
Right? I spent quite a while trying to figure out how to hack PHP to mock these built in functions. This was like a dream!!